x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -255,5 +251,80 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
         },
       },
     },
+    running: {
```

Comment:
let's delete `running`, we are not using it and too scare to make another call to SO. Let's wait and see

Reply:
This was going to be my question actually...if I create a rule, what is the status prior to the first run? Or if I disable it before its run?

Reply:
we were thinking that we can tell an approximation time when the rule is going to run. if you disable it before its run, we won't show anything, do you have a better idea?

Reply:
After chatting, we opted to simply show `--` if the rule has never run.  My initial concern was if I create a rule and see nothing in this column, I would wonder if the rule is starting or not.  So for this scenario, we'll also show the `--` but do a similar treatment as we do for [Stat loading indicator](https://elastic.github.io/eui/#/display/stat).  We felt this was the simplest solution at this time and still indicated to the user that something is happening with the rule after I create it.